migrations/5_amend_agreements.js

```diff
@@ -0,0 +1,42 @@
+require("babel-register");
+const controlAccounts = require("./accounts").default;
+
+const Neumark = artifacts.require("Neumark");
+const Commitment = artifacts.require("Commitment");
+
+// Maps roles to accounts on live network
+let PLATFORM_OPERATOR_REPRESENTATIVE;
+const RESERVATION_AGREEMENT =
+  "ipfs:QmPXME1oRtoT627YKaDPDQ3PwA8tdP9rWuAAweLzqSwAWT";
```

Comment:
I would take these hardcoded constants and the ones in accounts.js file and put them in separate file so they can be easily changed in future